### DISCUSSION TITLE
Option to group library mods together in the Mods tab

### DISCUSSION THREAD

**Discussion by G2Poti** [PrismLauncher/PrismLauncher#3232] [Ideas]
I often find myself trying to figure out which specific mods cause instabilities/crashes within my instances, but I don't always recognize the libraries as well as I do the actual mods themselves, so having the option to group them together (or at least filter by mod type) so I don't have to worry about accidentally disabling the ones I need would be very helpful.

(here I'm assuming that the main mods are much more likely to cause crashes than the libraries they depend on, but I could be very wrong as I'm just a casual player, in which case please ignore this post)

I think it could also look much neater to have them grouped and/or organized in that way, but that's just an opinion of course.

**Comment by DINEXXL** [comment]
I dream about it
